scrapers/kodi.py

```diff
@@ -0,0 +1,115 @@
+import sys
+import pathlib
+
+import mimetypes
+import base64
+
+import json
+import xml.etree.ElementTree as ET
+
+import py_common.graphql as graphql
+import py_common.log as log
+"""  
+This script parses kodi nfo files for metadata. The .nfo file must be in the same directory as the video file and must be named exactly alike.
+"""
+
+# If you want to ingest image files from the .nfo the path to these files may need to be rewritten. Especially when using a docker container.
+rewriteBasePath = False
+# Example: Z:\Videos\Studio_XXX\example_cover.jpg -> /data/Studio_XXX/example_cover.jpg
+basePathBefore = 'Z:\Videos'
+basePathAfter = "/data"
```

Comment:
I dont think we need that anymore since you added the base64 option
From what i understand we can either have a URL and thus we return it as a string
Or a full path (also from what i understand Kodi requires full path not relative ?) which we base64 encode

With that in mind i adjusted the code a bit and will it paste below

Reply:
Hm i didnt actually think of that....
You mean if the nfos were generated using a different os or setup....
You are probably right, your rewrite covers the case from windows  to linux or docker. What happens if its a linux -> linux or linux-> docker setup? It might need a bit of adjusting as the `.replace("\\", "/")` might replace something it shouldnt in that case